views/_rtm_rest_api_normal.njk

```diff
@@ -539,8 +539,8 @@ curl -X GET \
 
 Parameters | Optionality | Description
 ---|---|---
-limit | Optional | together with next to realize pagination, default on 10
-next | Optional | returned on the first query, the following query with this parameter to realize pagination
+limit | Optional | length of returned list,  together with next to implement pagination, default to 10
+next | Optional | returned on the first query, the following query with this parameter to implement pagination
```

Comment:
the following query with this parameter to implement pagination -> and the following query will use this parameter to implement pagination